tests/conftest.py

```diff
@@ -0,0 +1,19 @@
+def pytest_runtest_setup(item):
+    """
+    Per-test setup.
+    - Provide cache isolation incrementing the cache key prefix
+    """
+    increment_cache_key_prefix()
+
+
+def increment_cache_key_prefix():
```

Comment:
Why don't we use cache.clear()? These tests will never run on production, so there's no risk of loosing data by accident.